_includes/footer.html

```diff
@@ -1,66 +1,169 @@
 <footer class="usa-footer site-footer" role="contentinfo">
-  <div class="usa-identifier site-identifier">
-    <section class="usa-identifier__section usa-identifier__section--masthead" aria-label="Agency identifier">
-        <div class="usa-identifier__container">
-            <div class="usa-identifier__logos">
-                <a href="https://www.gsa.gov">
-                    {% image_with_class "_img/gsa-logo.svg" "usa-identifier__logo" "GSA logo identifier" %}
-                </a>
+    <div class="footer-section-bottom usa-footer__primary-section bg-accent-warm-light border-top border-accent-warm-dark">
```

Comment:
can you tell me why you went full utility classes here instead of sticking with the identifier classes? I can't tell what's better about this implementation.